website/common/script/content/spells.js

```diff
@@ -1,6 +1,6 @@
 import t from './translation';
 import each from 'lodash/each';
-import { NotAuthorized } from '../libs/errors';
+import { NotAuthorized, BadRequest } from '../libs/errors';
```

Comment:
can you remove this badrequest? It's not being used and causes the tests to fail :)

Reply:
Ah forgot about that, after I redid the check I stopped using it. It'll be gone in a moment...